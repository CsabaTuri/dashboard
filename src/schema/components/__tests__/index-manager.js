import React from "react"
import { shallow } from "enzyme"
import { shallowToJson } from "enzyme-to-json"

import IndexManager from "../index-manager"

it("display index components", () => {
  const comp = shallow(<IndexManager />)
  expect(shallowToJson(comp)).toMatchSnapshot()
})
