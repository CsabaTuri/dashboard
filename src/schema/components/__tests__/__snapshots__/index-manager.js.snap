exports[`test display index components 1`] = `
<Pivot>
  <PivotItem
    itemKey="index-info"
    linkText="Index Details">
    <Connect(IndexInfo) />
  </PivotItem>
  <PivotItem
    itemKey="index-delete"
    linkText="Delete">
    <Connect(Component) />
  </PivotItem>
</Pivot>
`;
